### PR TITLE
Creeper beam puzzle solver

### PR DESCRIPTION
**Pull request by msg-programs**
A rather fancy creeper beam solver. Doesn't always use the beam through the base, as there may be better options (that are also easier to hit with a bow from the ground). It could be forced to do that though, let me know if I should do that. 

Never mind the huge paragraph that once stood here, it's bugged, don't merge it yet.

### CODE REVIEW

**Comment by msg-programs** [comment]
Ok, tested it some more yesterday and today. It's good to go now. 
This took way longer than it should have...

**Review by kevinthegreat1 — APPROVED** [review]

File: src/main/java/de/hysky/skyblocker/config/SkyblockerConfig.java [hunk 1]
```diff
@@ -526,7 +526,10 @@ public static class Dungeons {
 		public boolean solveThreeWeirdos = true;
 		
 		@SerialEntry
-		public boolean blazesolver = true;
+		public boolean blazeSolver = true;
+
+        @SerialEntry
```
Comment (AzureAaron) [inline src/main/java/de/hysky/skyblocker/config/SkyblockerConfig.java:531]: Inconsistent formatting

File: src/main/java/de/hysky/skyblocker/config/categories/DungeonsCategory.java [hunk 2]
```diff
@@ -240,11 +240,19 @@ public static ConfigCategory create(SkyblockerConfig defaults, SkyblockerConfig
...
-								() -> config.locations.dungeons.blazesolver,
-								newValue -> config.locations.dungeons.blazesolver = newValue)
+						.name(Text.translatable("text.autoconfig.skyblocker.option.locations.dungeons.blazeSolver"))
+						.description(OptionDescription.of(Text.translatable("text.autoconfig.skyblocker.option.locations.dungeons.blazeSolver.@Tooltip")))
+						.binding(defaults.locations.dungeons.blazeSolver,
+								() -> config.locations.dungeons.blazeSolver,
+								newValue -> config.locations.dungeons.blazeSolver = newValue)
+						.controller(ConfigUtils::createBooleanController)
+						.build())
+                .option(Option.<Boolean>createBuilder()
```
Comment (AzureAaron) [inline src/main/java/de/hysky/skyblocker/config/categories/DungeonsCategory.java:250]: Inconsistent formatting

**Review by AzureAaron — APPROVED** [review]
Small changes, could be done afterwards if the solver doesn’t want to be delayed further

**Review by kevinthegreat1 — APPROVED** [review]

**Review by AzureAaron — APPROVED** [review]

**Pull request merged**